modules/webgl/src/classes/texture-cube.js

```diff
@@ -25,31 +25,7 @@ export default class TextureCube extends Texture {
 
   /* eslint-disable max-len, max-statements */
   initialize(props = {}) {
-    const {/* format = GL.RGBA, */ mipmaps = true, parameters = {}} = props;
-
-    // let {width = 1, height = 1, type = GL.UNSIGNED_BYTE, dataFormat} = props;
-
-    // Deduce width and height based on one of the faces
```

Comment:
I recall there were lots of problems caused by commenting out this code but at the time we couldn't get this code path to work with glTF.

I don't have all the subtle issues fresh but at least I think that not having it here makes Texture2D and TextureCube behave quite differently.

Since we don't use TextureCubes much it doesn't really matter, this can be reimplemented once we do.